packages/components/form/custom-elements.json

```diff
@@ -453,7 +462,7 @@
                 {
                   "name": "message",
                   "type": {
-                    "text": "string"
+                    "text": "string | Promise<string>"
```

Comment:
It looks like in the documentation website it's not showing `Promise<string>` properly, it only shows `Promise`, but maybe it's sth that we still need to check maybe within another issue?
<img width="961" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/40c7b19e-c274-4ea2-8569-a6932d19cb4e">

Reply:
I believe this may be related to the PR @Diaan created yesterday on `custom-elements.json` in the website not updating.